src/libraries/System.Runtime.InteropServices.JavaScript/gen/JSImportGenerator/JSImportGenerator.csproj

```diff
@@ -33,7 +33,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.InteropServices\gen\Microsoft.Interop.SourceGeneration\Microsoft.Interop.SourceGeneration.csproj" Pack="true" PackagePath="analyzers/dotnet/cs" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.InteropServices\gen\Microsoft.Interop.SourceGeneration\Microsoft.Interop.SourceGeneration.csproj" />
+    <AnalyzerReference Include="..\Microsoft.Interop.SourceGeneration\Microsoft.Interop.SourceGeneration.csproj" Pack="true" ReferenceAnalyzer="false" />
```

Comment:
I debated leaving this stuff alone, but decided against it since this project has other packable properties set like `IncludeBuildOutput`, `SuppressDependenciesWhenPacking`, etc. I didn't want someone to have to waste time rediscovering this problem if they turn this on in the future.  So I made it so that this project would pack correctly if you pass in `IsPackable=true`.

Alternatively we can delete all the packaging properties if we don't think we'll ever want to pack these.  cc @elinor-fung @AaronRobinsonMSFT

Reply:
I think we should delete all the packaging properties. We did have aspirations for a down-level story, but I don't think that is really tenable or necessary anymore. The `Microsoft.Interop.SourceGeneration` is possible but at this point the `LibraryImport` and `JSImport` source generators are in-box forever.

Reply:
Ok, let me do this.